libs/wire-api/src/Wire/API/MLS/CommitBundle.hs

```diff
@@ -31,14 +41,40 @@ data CommitBundle = CommitBundle
   }
   deriving (Eq, Show)
 
-instance ParseMLS CommitBundle where
-  parseMLS = CommitBundle <$> parseMLS <*> parseMLSOptional parseMLS <*> parseMLS
+instance ConvertProtoLens Proto.Mls.CommitBundle CommitBundle where
+  fromProtolens protoBundle = CP.label "CommitBundle" $ do
+    CommitBundle
+      <$> CP.label "commit" (decodeMLS' (view Proto.Mls.commit protoBundle))
+      <*> CP.label
+        "welcome"
+        ( let bs = view Proto.Mls.welcome protoBundle
+           in if BS.length bs == 0
+                then pure Nothing
+                else Just <$> decodeMLS' bs
+        )
+      <*> CP.label "group_info_bundle" (fromProtolens (view Proto.Mls.groupInfoBundle protoBundle))
+  toProtolens bundle =
+    let commitData = rmRaw (cbCommitMsg bundle)
+        welcomeData = maybe mempty rmRaw (cbWelcome bundle)
```

Comment:
`maybe mempty = foldMap`. But this is fine, of course.

Reply:
We could add that to hlint, what do you think?